### ISSUE TITLE
New issue but it's an old one.

### ISSUE THREAD

**Issue by Rdgeno** [rapid7/metasploit-framework#12649]
## Steps to reproduce

How'd you do it?

1. ...msfconsole 
2. ...armitage sudo -E armitage or armitage ./armitage and sudo -E ./armitage aren't recognized even though people said to try it.

I keep getting I can not find a database.yml file. I really need it. about two hours in I found a post that explained that service metasploit start/stop doesn't work anymore. 
I'm about six hours into this now and I'm all out of places to look for answers on Google and things to try. 
Metasploit worked fine until I upgrade to the new version of Kali. I would just reinstall 2019.3 but by some miracle of God I actually got John the Ripper to use my AMD GPU I kind of lost track of what I was trying so I don't know how I did it and that was two months of trying till I got it to work, Kali doesn't like AMD. 
The yml file is there. Metasploit has always been so easy to use and reliable. This problem is all over the place on Google I even found the issue posted here. There is nothing on Google to fix this. 
I'm just letting you know. I don't know what else to say it's broke.I'm sorry to say it.

msfdb init ,
Metasploit running on Kali Linux as root, using system database
A database appears to be already configured, skipping initialization
The only error I get is the yaml and to move the Try setting the MSF_DATABASECONFIG to a file that exist. 
I found a couple places saying they solved it and followed what they said. I keep getting the same window saying the same thing. 
I just used this on Wednesday before the upgrade it worked perfect.
I even saw that you guys where discussing this but the issue was closed without an answer, I think it was you guys I know there where at least two on Git. I think there was a link I one I followed it if there was and needless to say no resolution.
I don't want to blow out the screen with everything I tried if you want I can copy my search logs and send them to you there is honestly nothing I haven't tried. 
Hope you guys can get her back up and running. This has always been tied with one other tool as my favorite one of the reasons is it's not like all the others it always works. 

## Expected behavior For Armitage to open and Metasploit to work like it always has.
You might also want to check the last ~1k lines of
`/opt/metasploit/apps/pro/engine/config/logs/framework.log` or
`~/.msf4/logs/framework.log` for relevant stack traces 

There's a lot of stuff in those I don't know what you will or wont want. I don't want to nail you with a 12 screen post. If you want them let me know I'm more than happy to put them up.

### Metasploit version

Get this with the `version` command in msfconsole (or `git log -1 --pretty=oneline` for a source install).

### I installed Metasploit with:
- [ ] Kali package via apt Yes
- [ ] Omnibus installer (nightly) Yes
- [ ] Commercial/Community installer (from http://www.rapid7.com/products/metasploit/download.jsp) Every version I could find I tried. Every installer.
- [ ] Source install (please specify ruby version) (v1:2.5.2) Then when I input ruby -v I get.

ruby -v
ruby 2.5.7p206 (2019-10-01 revision 67816) [x86_64-linux-gnu]

What OS are you running Metasploit on? Kali Linux 2019.4

**Comment by wvu-r7** [comment]
Armitage is neither created nor supported by us. Please see http://www.fastandeasyhacking.com/contact.

**Comment by Rdgeno** [comment]
Glad I saw your reply in my email solved it.
Incase anyone happens on this. The problem I had as of November 2019 isnt caused by Metasploit or Armitage its caused buy pentesters framework and or upgrading from Kali 2019.3 to .4 either one causes the problem. Either on it's own breaks it. So dont upgrade Kali clean install and dont install Pentesters Framework that thing wrecked a lot of stuff.
Thanks for your reply guys sorry I bothered you.
